test/random.jl

```diff
@@ -370,11 +372,11 @@ for rng in ([], [MersenneTwister(0)], [RandomDevice()])
     for f! in [rand!, randn!, randexp!]
         for T in (f! === rand! ? types : f! === randn! ? cftypes : ftypes)
             X = T == Bool ? T[0,1] : T[0,1,2]
-            for A in (Array{T}(5), Array{T}(2, 3))
+            for A in [Array{T}(5), Array{T}(2, 3), GenericArray{T}(5), GenericArray{T}(2, 3)]
```

Comment:
Ah yes sure... so in this case I get `Array{AbstractArray{Int64,1},1}`, and each item keeps its separate type, though I can change it back to a tuple if this is prefered.